### PR TITLE
Create facebook-pixel.js

### PR DESCRIPTION
**Pull request by isantolin**
Adds support to Facebook Pixel tracking

### CODE REVIEW

File: src/facebook-pixel/facebook-pixel.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * This feature enables Facebook Pixel to send certain events, such as play, pause, ended, etc. It requires previous configuration
+ * on Pixel to send events properly.
+ * @see https://developers.facebook.com/docs/facebook-pixel
+ */
+
+// Feature configuration
+
+Object.assign(mejs.MepDefaults, {
+	/**
+  * @type {String}
```
Comment (rafa8626) [inline src/facebook-pixel/facebook-pixel.js]: Formatting; please align comments

File: src/facebook-pixel/facebook-pixel.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(_dereq_,module,exports){
```
Comment (rafa8626) [inline src/facebook-pixel/facebook-pixel.js]: This is not necessary; please remove this.

**Comment by rafa8626** [comment]
Just need this file; create a npm task inside the package.json for this feature; see the other tasks for reference.

**Comment by isantolin** [comment]
@rafa8626 Ready to merge

**Comment by rafa8626** [comment]
PR merged. Thanks. Please star both https://github.com/mediaelement/mediaelement and https://github.com/mediaelement/mediaelement-plugins to continue improving the package and giving it more popularity; also tell your friends/coworkers/etc. that have a GitHub account to do the same to support the project, if you don't mind. It would be greatly appreciated it.

**Pull request merged**